Tests/SwiftProtobufTests/Test_JSON.swift

```diff
@@ -481,6 +482,20 @@ class Test_JSON: XCTestCase, PBTestHelpers {
         }
     }
 
+    func testSingleDouble_roundtrip() throws {
+        for _ in 0..<10000 {
+            let d = drand48()
+            assertRoundTripJSON {$0.singleDouble = d}
```

Comment:
If these were to fail, will be be able to tell what the value was?  i.e. - is the only logging by way of TextFormat or something that would be going though the formatting code?